### PR TITLE
Added possibility to Post a Join Channel Message

### LINKED ISSUE #64: Add Channel Join Message
**Is your feature request related to a problem? Please describe.**
Without the logging Flag on Info, i don't know if/when the Client successfully connected to the Twitch Channel.

**Describe the solution you'd like**
Toggleable and Customizable Channel Join Message

**Describe alternatives you've considered**
Adding a ChannelJoin Log to Warning Level?

**Additional context**
Don't have any

### PR DESCRIPTION
**Pull request by Locxion**
Added OnChannelJoined Event to Client
Added PostJoinMessage bool to ConnectionConfig
Added PostJoinMessage string to ConnectionConfig

closes #64 

Had to close #63 and reopen cause of f*ck up in my repo

### CODE REVIEW

File: Runtime/ConnectionManager.cs [hunk 1]
```diff
@@ -108,6 +107,15 @@ public class ConnectionManager {
             Logger.LogError("Invalid bot login, need to re-authenticate");
         }
 
+        private void ClientOnOnJoinedChannel(object sender, OnJoinedChannelArgs e)
```
Comment (pilky) [inline Runtime/ConnectionManager.cs:110]: You've got a duplicate `On` in the method name

File: Runtime/ConnectionManager.cs [hunk 2]
```diff
@@ -108,6 +107,15 @@ public class ConnectionManager {
             Logger.LogError("Invalid bot login, need to re-authenticate");
         }
 
+        private void ClientOnOnJoinedChannel(object sender, OnJoinedChannelArgs e)
+        {
```
Comment (pilky) [inline Runtime/ConnectionManager.cs:111]: Just to keep formatting consistent in the code, opening brackets go at the end of the line

File: Runtime/TwitchManager.cs [hunk 3]
```diff
@@ -59,6 +59,8 @@ public class TwitchManager : MonoBehaviour {
             ClientID = "",
             ChannelName = "",
             BotName = "",
+            PostConnectMessage = true,
```
Comment (pilky) [inline Runtime/TwitchManager.cs:62]: I would be inclined to default this to `false` so it doesn't unexpectedly appear for existing users of Twitchmata when they upgrade

**Review by pilky — COMMENTED** [review]
Hadn't consider this but it's definitely a useful additions. Just a few small changes needed then I'll approve and merge this. Thanks for the PR!

**Comment by Locxion** [comment]
All done 👍

**Review by pilky — APPROVED** [review]

**Comment by pilky** [comment]
@Locxion Sorry for taking so long to finally approve and merge this! Life has taken over the past few months so have hardly got any code stuff done, be that my own projects or work stuff

**Comment by Locxion** [comment]
I know how it is ... im there right now too and cant get anything else done ... sleep work eat repeat ... no worries

**Pull request merged**